libraries/cyclestreets-view/src/main/java/net/cyclestreets/tiles/TileSource.java

```diff
@@ -25,6 +25,7 @@
 public class TileSource {
 
   private static final String TAG = Logging.getTag(TileSource.class);
+  private static int boxHeight;
```

Comment:
Why is the box - conceptually unique to the `LiveRideOverlay` - associated with the `TileSource`? That doesn't seem to make logical architectural sense.

Reply:
I don't know if you saw my email about this - what would be the best way to do this?  Should I create a separate class or is there a better place to put it which is accessible to ControllerOverlay?

Reply:
I've renamed it attributionUpShift :-)